### PR TITLE
Dashboard UI implementation

### PR DESCRIPTION
**Pull request by lin8231**
* To be populated with real data in the next PR

### CODE REVIEW

File: project/src/test/java/com/google/sps/servlets/AuthServletTest.java [hunk 1]
```diff
@@ -1,80 +1,80 @@
-// Copyright 2021 Google LLC
```
Comment (texm) [inline project/src/test/java/com/google/sps/servlets/AuthServletTest.java:1]: I think we should exclude changes to this file in this PR

**Review by texm — APPROVED** [review]
LGTM, but remove AuthServletTest changes :)

File: project/src/test/java/com/google/sps/servlets/AuthServletTest.java (hunk 1 above)
Comment (lin8231) [reply project/src/test/java/com/google/sps/servlets/AuthServletTest.java:1]: My bad, that was a mistake

Comment (lin8231) [reply project/src/test/java/com/google/sps/servlets/AuthServletTest.java:1]: Fixed

**Review by Phoebe-Khokgawe — APPROVED** [review]

**Pull request merged**